### ISSUE TITLE
windows support?

### ISSUE THREAD

**Issue by Jorgen-VikingGod** [chuckfairy/node-webcam#1]
I'm searching for a real cross platform support library - but all of them did not have windows support...
Ok I'm normally workign on OSX and Linux - but for my current project; I want also to run it on windows for my friends.

Currently I'm using ffmpeg and spawn - to get the image - but it is not the best way :-1: 
Hope some of you guys have a better idea.

```
var spawn = require('child_process').spawn;

function captureWebcamImage(w, h) {
  var video_size = w + 'x' + h; //hd720
  var ffmpegArgs = ['-y', '-f', 'vfwcap', '-video_size', video_size, '-i', 'pipe:0', '-vframes', '1', './public/last_capture.jpg'];
  ffmpeg = spawn('./ffmpeg', ffmpegArgs);
  ffmpeg.on('error', function(err) {
	  console.log('an error happened: ' + err.message);
  });
  ffmpeg.on('exit', function() {
    console.log('new image captured');
  });
}
var interval = setInterval(function(){captureWebcamImage(1280, 720);}, 5000);
```

**Comment by chuckfairy** [comment]
Total cross platform is the goal of this project. I haven't had much luck getting the webcam working in the Windows VBox, but I'll get a real Windows machine running for this project.

I plan on implementing something like this https://batchloaf.wordpress.com/2011/04/06/snapz-a-command-line-cam-image-grabber/ . Building that into an npm install shouldn't be too hard. Duly noted on the FFMPEG. Thanks for your input.

**Comment by Jorgen-VikingGod** [comment]
great to know :+1: 
I also try out the tools from the blog you posted;
but they can only grab screens with 640x480 pixel resolution.
This was the reson I use the ffmpeg...
Maybe you can use the DirectDraw API instead of Win32 API to capture image...

**Comment by chuckfairy** [comment]
I have added in partial windows support. I found a library using the DirectShow API and have integrated into node-webcam (link to binding below). I'm not entirely sure if that's the same thing as the DirectDraw (Windows newbie here), but I would love your thoughts on the current build. So far missing some options that are included in other webcams. Thanks

https://github.com/chuckfairy/node-webcam/tree/master/src/bindings/CommandCam
